packages/js/src/session/session.ts

```diff
@@ -18,24 +20,64 @@ export class Session {
   }
 
   public get applicationIdentifier() {
-    return this.#options.applicationIdentifier;
+    if ('applicationIdentifier' in this.#options) {
+      return this.#options.applicationIdentifier;
+    }
+
+    return undefined;
   }
 
   public get subscriberId() {
-    return this.#options.subscriber.subscriberId;
+    if ('subscriber' in this.#options && this.#options.subscriber) {
+      return this.#options.subscriber.subscriberId;
+    }
+
+    return undefined;
+  }
```

Comment:
Same here, feels like un needed if statement

Reply:
it was needed before when the type was the union was with type KeylessInitializeSessionArgs =  {}
same as with other cases it was solved with ` {} & { [K in string]?: never }`, now it is more clean

![image](https://github.com/user-attachments/assets/56fc9fa6-3421-495f-8022-4a0622f4b8ee)